.sentryclirc

```diff
@@ -0,0 +1,4 @@
+[defaults]
+url = https://sentry.integreat-app.de/
+org = tur-an-tur-digitalfabrik
```

Comment:
is that correct? shouldn't it be `tuer-an-tuer-digitalfabrik`?

Reply:
Unfortunately not :P

Reply:
Is is correct unfortunately